dex-auth/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - jlewi
```

Comment:
I shouldn't be listed as an explicit approver here since I don't have specific context.
Does the same hold for @kkasravi ?

Alphabetical order please.